sass/all.scss

```diff
@@ -0,0 +1,17 @@
+// Vendor
```

Comment:
Vendor was to indicate which things are third-party vs. not -- probably could just remove this comment.

Guess it's fine to uses scss instead of sass, if that's what you prefer.

Reply:
css/scss tooling (prettier) seems to not want to use sass but is cool with css/scss. Its just a minor code adjustment (brackets and semicolons).

Reply:
Oh, interesting! I hadn't tried to use prettier on Sass... thanks for transitioning it.